src/event/event_kevent.c

```diff
@@ -614,6 +631,8 @@ _dispatch_kq_poll(dispatch_wlh_t wlh, dispatch_kevent_t ke, int n,
 		}
 		r = kevent_qos(kqfd, ke, n, ke_out, n_out, buf, avail, flags);
 #else
+		(void)buf; // buf is unused here
```

Comment:
remove the comment, `(void)variable` means it's unused throughout the repository already